packages/a/angular.js.json

```diff
@@ -18,13 +17,16 @@
     "type": "git",
     "url": "git://github.com/angular/angular.js.git"
   },
-  "npmName": "angular",
-  "npmFileMap": [
-    {
-      "basePath": "",
-      "files": [
-        "angular.*"
-      ]
-    }
-  ]
+  "autoupdate": {
+    "source": "npm",
+    "target": "angular",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "angular.*"
```

Comment:
This won't grab `angular-csp.css`?

Reply:
Interesting, I didn't check the fileMap since I assumed it was correct in the previous version.